File: 240411/부분 문자열의 존재 유무/presence-of-substring.py
ee=False
ea=False
str=input()
for i in range(0,len(str)-1):
    if(str[i]=='e'):
        if(str[i+1]=='e'):
            ee=True
        elif(str[i+1]=='a'):
            ea=True
if(ee):
    print('YES',end=' ')
else:
    print('NO',end=' ')
if(ea):
    print('YES')
else:
    print('NO')

File: 240914/부분 문자열의 존재 유무/presence-of-substring.kt
fun main()
{
    val str=readLine()!!
    val len=str.length
    var ee=false
    var ea=false
    for(i in 0..len-2)
    {
        if(str[i]=='e')
        {
            if(str[i+1]=='e')ee=true
            if(str[i+1]=='a')ea=true
        }
    }
    if(ee==true)print("YES ")
    else print("NO ")
    if(ea==true)print("YES")
    else print("NO")
}